pulp_rpm/app/models/package.py

```diff
@@ -310,7 +310,14 @@ def createrepo_to_dict(cls, package):
             changelog_limit = settings.KEEP_CHANGELOG_LIMIT or 1
             # changelogs are listed in chronological order, grab the last N changelogs from the list
             changelogs = changelogs[-changelog_limit:]
-
+        files = getattr(package, CR_PACKAGE_ATTRS.FILES, [])
+        seen = set()
+        for path in files:
+            if path in seen:
+                log.warn(f"Duplicated file: {path})")
```

Comment:
We don't need to warn more than once per package.  I'd also make the message a bit more specific, maybe something like:  "Package {} lists some files more than once"